cw20/contracts.js

```diff
@@ -1877,6 +1877,11 @@ module.exports = {
       name: "GarudaX Token",
       icon: "https://i.ibb.co/jkwPpnS/zz1.png",
     },
+    terra1g6fm3yu79gv0rc8067n2nnfpf0vks6n0wpzaf4u7w48tdrmj98zsy7uu00: {
+      protocol: "Terra Tritium",
+      name: "Tritium Token",
```

Comment:
@andrexbass After querying the corresponding smart contract, the name is returned as `Terra Tritium Token`.  You will need to update either the name on the smart contract or the name which is provided here so that these two values match.

Reply:
Done.